bot.py

```diff
@@ -129,6 +149,15 @@ def translate(self, text: str):
         """
         for key, value in self.substitutions.items():
             text = text.replace(key, value)
+        i = 0
+        while i < len(text):
+            if(text[i] == '.'):
+                if(randint(0, 5) == 0):
+                    seed(i) # random is not very random apparently
+                    randSuffix = self.suffixes[randint(0, len(self.suffixes)-1)]
+                    text = text[:i] + randSuffix + text[i + 1:]
+            i += 1
+
```

Comment:
To clarify: This portion of the code will look for full stops and add a suffix to it? Should it also include `!` and `?`
Also, we should probably do regex boundaries less `...` will turn into some disgusting abomination

Reply:
Ah I wasn't too sure what this part did. Thanks for the clarification

Reply:
Yep you nailed it, it does it randomly 1/5 of the time though and not for every occurence. Should have added a comment explaining this bit of code, I'll add one now. I'll also include `!` and `?`, as well as regexing for punctuation blocks.